js/w3c/style.js

```diff
@@ -73,15 +73,16 @@ define(
             case "LC-NOTE":
               styleFile += "WD";
               break;
+            case "WG-NOTE":
             case "FPWD-NOTE":
-              styleFile += "WG-NOTE";
+              styleFile += "WG-NOTE.css";
```

Comment:
Hmm - yes, you are:

![screenshot 2016-04-19 at 22 01 12](https://cloud.githubusercontent.com/assets/520688/14662281/4521052c-067a-11e6-8760-25fea21ac043.png)